AppManifest.json

```diff
@@ -34,10 +34,10 @@
             ],
             "dapr": {
                 "cli": {
-                    "version": "1.9.1"
+                    "version": "1.10.0"
                 },
                 "runtime": {
-                    "version": "1.8.1"
+                    "version": "1.10.2"
```

Comment:
Change can be risky :) Don't merge until full manual test :)

Reply:
we already have this in all other repos, so the risk should be minimal since we tested already a lot ;)

Reply:
Unfortunately example "SeatAdjuster" doesn't work for me"
Looks like issue with Feeder Can:
```
v0.1.1: Pulling from eclipse/kuksa.val.feeders/dbc2val
5c5ad4a052fa: Pull complete 
fbe98669be7e: Pull complete 
3c308995c66b: Pull complete 
3aee17f80ab1: Pull complete 
ec263aca9c06: Pull complete 
Digest: sha256:0520da25c1359acfd08edbfb0e9fa6f99840dca9286465af8ce8d8c60eddceca
Status: Downloaded newer image for ghcr.io/eclipse/kuksa.val.feeders/dbc2val:v0.1.1
2023-04-17 07:12:35,834 INFO dbcfeeder: Using config: config/dbc_feeder.ini
2023-04-17 07:12:35,837 INFO dbcfeeder: Starting CAN feeder
Traceback (most recent call last):
  File "dbcfeeder.py", line 530, in <module>
  File "dbcfeeder.py", line 446, in main
  File "dbcfeeder.py", line 111, in start
  File "dbc2vssmapper.py", line 32, in __init__
FileNotFoundError: [Errno 2] No such file or directory: '/data/mappingDefault.yml'
[8] Failed to execute script 'dbcfeeder' due to unhandled exception!
```

Reply:
and the app should also be runnable without the feeder...

Reply:
as mentioned, please raise a new issue, since this is unrelated to this PR